TKGame/Player.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Diagnostics;
+using Microsoft.VisualBasic;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Media;
+
+namespace TKGame
+{
+    class Player : Entity
+    {
+        private static Player instance;
+        public static Player Instance
+        {
+            get
+            {
+                // Creates the player if it doesn't already exist
+                if (instance == null)
+                    instance = new Player();
+
+                return instance;
+            }
+        }
```

Comment:
Singletons are typically considered an anti-pattern, but they're pretty handy in game dev since there's quite a few scenarios where we only expect one instance of an object. If you wanna improve on this, there's a way to make this thread-safe with `lock()`